examples/frompapers/Brunel_Wang_2001.py

```diff
@@ -0,0 +1,222 @@
+P_I.v = V_L
+
+eqs_glut = '''
+s_NMDA_tot_post = w * s_NMDA : 1 (summed)
+ds_NMDA / dt = - s_NMDA / tau_NMDA_decay + alpha * x * (1 - s_NMDA) : 1 (clock-driven)
+dx / dt = - x / tau_NMDA_rise : 1 (clock-driven)
+w : 1
+'''
+
+eqs_pre_glut = '''
+s_AMPA += w
+x += 1
+'''
+
+eqs_pre_gaba = '''
+s_GABA += 1
+'''
+
+eqs_pre_ext = '''
+s_AMPA_ext += 1
+'''
+
+# E to E
+C_E_E = Synapses(P_E, P_E, model=eqs_glut, on_pre=eqs_pre_glut)
+C_E_E.connect('i != j')
+C_E_E.w[:] = 1
+
+for pi in range(N_non, N_non + p * N_sub, N_sub):
+
+    # internal other subpopulation to current nonselective
+    C_E_E.w[C_E_E.indices[:, pi:pi + N_sub]] = w_minus
+
+    # internal current subpopulation to current subpopulation
+    C_E_E.w[C_E_E.indices[pi:pi + N_sub, pi:pi + N_sub]] = w_plus
+
+# E to I
+C_E_I = Synapses(P_E, P_I, model=eqs_glut, on_pre=eqs_pre_glut)
+C_E_I.connect()
+C_E_I.w[:] = 1
+
+# I to I
+C_I_I = Synapses(P_I, P_I, on_pre=eqs_pre_gaba)
+C_I_I.connect('i != j')
+
+# I to E
+C_I_E = Synapses(P_I, P_E, on_pre=eqs_pre_gaba)
+C_I_E.connect()
+
+# external noise
+C_P_E = PoissonInput(P_E, 's_AMPA_ext', C_ext, rate, '1')
+C_P_I = PoissonInput(P_I, 's_AMPA_ext', C_ext, rate, '1')
+
+# at 1s, select population 1
+C_selection = int(f * C_ext)
+rate_selection = 25 * Hz
+stimuli1 = TimedArray(np.r_[np.zeros(40), np.ones(2), np.zeros(100)], dt=25 * ms)
+input1 = PoissonInput(P_E[N_non:N_non + N_sub], 's_AMPA_ext', C_selection, rate_selection, 'stimuli1(t)')
+
+# at 2s, select population 2
+stimuli2 = TimedArray(np.r_[np.zeros(80), np.ones(2), np.zeros(100)], dt=25 * ms)
+input2 = PoissonInput(P_E[N_non + N_sub:N_non + 2 * N_sub], 's_AMPA_ext', C_selection, rate_selection, 'stimuli2(t)')
+
+# at 4s, reset selection
+stimuli_reset = TimedArray(np.r_[np.zeros(120), np.ones(2), np.zeros(100)], dt=25 * ms)
+input_reset_I = PoissonInput(P_E, 's_AMPA_ext', C_ext, rate_selection, 'stimuli_reset(t)')
+input_reset_E = PoissonInput(P_I, 's_AMPA_ext', C_ext, rate_selection, 'stimuli_reset(t)')
+
+# monitors
+N_activity_plot = 15
+sp_E_sels = [SpikeMonitor(P_E[pi:pi + N_activity_plot]) for pi in range(N_non, N_non + p * N_sub, N_sub)]
+sp_E = SpikeMonitor(P_E[:N_activity_plot])
+sp_I = SpikeMonitor(P_I[:N_activity_plot])
+
+r_E_sels = [PopulationRateMonitor(P_E[pi:pi + N_sub]) for pi in range(N_non, N_non + p * N_sub, N_sub)]
+r_E = PopulationRateMonitor(P_E[:N_non])
+r_I = PopulationRateMonitor(P_I)
+
+# simulate, can be long >120s
+net = Network(collect())
+net.add(sp_E_sels)
+net.add(r_E_sels)
+net.run(4 * second, report='stdout')
+
+# plotting
+title('Population rates')
+xlabel('ms')
+ylabel('Hz')
+
+plot(r_E.t / ms, r_E.smooth_rate(width=25 * ms) / Hz, label='nonselective')
+plot(r_I.t / ms, r_I.smooth_rate(width=25 * ms) / Hz, label='inhibitory')
+
+for i, r_E_sel in enumerate(r_E_sels[::-1]):
+    plot(r_E_sel.t / ms, r_E_sel.smooth_rate(width=25 * ms) / Hz, label='selective {}'.format(p - i))
+
+legend()
+show()
```

Comment:
Please replace `show()` by `figure()` so that the two plots are visible at the same time.